src/promptflow/promptflow/contracts/flow.py

```diff
@@ -870,8 +880,13 @@ def get_connection_names(self):
 
         return set({item for item in connection_names if item})
 
-    def get_connection_input_names_for_node(self, node_name):
-        """Return connection input names."""
+    def get_connection_input_names_for_node(self, node_name, return_unassigned_inputs=False):
```

Comment:
Do we really need the scenario that only returns unassigned inputs?

Reply:
Offline synced, let me directly modify this function